haskell/stack.yaml

```diff
@@ -11,11 +11,11 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    37449aa1c4e889557058e6895822fa97e5a0215a # master
+    895a659d9faf2f32ecdd11fe39ee8edd4d09146f # master
```

Comment:
Just in case: https://gitlab.com/morley-framework/morley/-/merge_requests/532 should be merged first and this reference should be updated to actually point to master.

Reply:
Yup, I'll try to get !532 reviewed and merged asap.

Reply:
On that note, do you know if that's the reason why the [pipeline build is failing](https://buildkite.com/serokell/stablecoin/builds/735#7e0e785e-0b91-4eb7-81a0-a07bfcbf1445/63-90)? 

It's weird that `stack` builds this just fine locally, regardless of whether the commit exists in master or not, but it fails in the pipeline...

Reply:
Oh, yes, that's a known problem previously discussed in this thread: https://serokell.slack.com/archives/CUMAQ3PL4/p1592934064227100. And it looks like the problem is quite deep :(